ctk/common/src/main/kotlin/org/codice/compliance/verification/core/requests/CoreLogoutRequestProtocolVerifier.kt

```diff
@@ -24,8 +24,8 @@ class CoreLogoutRequestProtocolVerifier(private val samlRequestDom: Node) :
 
     /** 3.7.1 Element <LogoutRequest>*/
     override fun verify() {
-        super.verify()
         verifyLogoutRequest()
+        super.verify()
```

Comment:
Made this change because of the comment above about other verification running before the `CommonDataTypeVerifier`